src/normalize/entries/index.js

```diff
@@ -0,0 +1,25 @@
+const { getDecodedPostDataParamEntries } = require('./decodePostDataParams')
+const { getSortedEntries } = require('./sortEntries')
+
+/**
+ * Validate whether archive has `entries` property
+ * @param {HAR} archive
+ * @return {boolean} If true `entries` property is present
+ */
+function doesArchiveHaveEntries(archive) {
+  return !!archive && !!archive.log && !!archive.log.entries
+}
+
+/**
+ * Normalizes archive entries
+ * @param {Entry[]} entries
+ * @param {{ addSleep?: boolean }} options
+ * @returns {Entry[]} entries
+ */
+function getNormalizedEntries(entries, options) {
+  entries = getSortedEntries(entries, options)
+  entries = getDecodedPostDataParamEntries(entries)
+  return entries
+}
```

Comment:
Yeaaaah, agree with reassignment 👍
 
The `getSortedEntries` is probably more fitting than `setSleepTimings` or `createSleepStatements`. It will always sort entries if there's a `startedDateTime` available, but only add `sleep` properties if `{ addSleep: true }` option is passed.

Reply:
Fair enough 👍 
It would made sense to split the two to separate the concerns then.
```
sortEntries
decodePostdataParams

if (option.addSleep) {
   return withSleep(..)
}

return result
```